### PR TITLE
Reporting variables

### PR DESCRIPTION
**Pull request by truthfool**
## Description
- added reporting variables
- Correction in accountHolding Institution Id values

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Comment by truthfool** [comment]
@fynmanoj @ankita10r

File: helm/ph-ee-engine/templates/connector-mpesa.yaml [hunk 1]
```diff
@@ -125,7 +125,7 @@ spec:
             - name: "ACCOUNTS_ROSTER_CURRENCY"
               value: "{{ .Values.mpesa.paybill.roster.currency }}"
             - name: "ACCOUNT_HOLDING_INSTITUTION_ID"
-              value: "{{ .Values.mpesa.paybill.accountHoldingInstitutionId }}"
+              value: "{{ .Values.mpesa.paybill_accountHoldingInstitutionId }}"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/templates/connector-mpesa.yaml:128]: if you are giving this as env variable  dot (.) is fine.  keep it as  .Values.mpesa.paybill.accountHoldingInstitutionId

File: helm/ph-ee-engine/values.yaml [hunk 2]
```diff
@@ -442,8 +442,8 @@ mpesa:
       client_key: "0pLxbN83FrOl5Nd0Fh9Zi5BQlMxSL2n5"
       client_secret: "YzuGNoJxeub8ZC6d"
       pass_key: "bfb279f9aa9bdbcf158e97dd71a467cd2e0c893059b10f78e6b72ada1ed2c919"
+  paybill_accountHoldingInstitutionId: "default"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/values.yaml:445]: keep this as . format and make sure that this variable is added to the corresponding service

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/ph-ee-engine/templates/connector-mpesa.yaml (hunk 1 above)
Comment (fynmanoj) [reply helm/ph-ee-engine/templates/connector-mpesa.yaml:128]: - name: "paybill_accountHoldingInstitutionId"
   value: "default"
            add these in values.yml as a workaround

**Pull request closed without merging**